openapi/platform/platform.yaml

```diff
@@ -2277,6 +2277,273 @@ paths:
+        in: path
+        required: true
+        example: U-12345678-abcd-1234-abcd-123456789abc
+        description: The ID of the Custom Fields - actually the same id of the User
+    patch:
+      summary: Update Custom Fields
+      operationId: patch-userCustomFields-by-id
+      requestBody:
+        content:
+          application/vnd.api+json:
+            schema:
+              type: object
+              properties:
+                data:
+                  $ref: '#/components/schemas/userCustomFields'
+              required:
+                - data
+            examples:
+              'Example with String, Date and Integer':
+                value:
+                  data:
+                    id: U-12345678-abcd-1234-abcd-123456789abd
+                    type: userCustomFields
+                    attributes:
+                      partnerFields:
+                        - fieldID: FieldID-123448ca-abc0-abc5-abc2-abc640bf4b99
+                          integerValue: 1000
+                        - fieldID: FieldID-987448ca-abc0-abc5-abc2-abc640bf4b99
+                          dateValue: A good date value
+                        - fieldID: FieldID-abc448ca-abc0-abc5-abc2-abc640bf4b99
+                          stringValue: A good string value
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/userCustomFields'
+              examples:
+                Return Example:
+                  value:
+                    data:
+                      id: U-12345678-abcd-1234-abcd-123456789123
+                      type: userCustomFields
+                      attributes:
+                        partnerFields:
+                          - fieldID: FieldID-123448ca-abc0-abc5-abc2-abc640bf4b99
+                            integerValue: 1000
+                            title: Title of 123448ca field
+                            description: Description for 123448ca field
+                            fieldType: integer
+                          - fieldID: FieldID-987448ca-abc0-abc5-abc2-abc640bf4b99
+                            dateValue: A good date value
+                            title: Title of 987448ca field
+                            description: Description for 987448ca field
+                            fieldType: date
+                          - fieldID: FieldID-abc448ca-abc0-abc5-abc2-abc640bf4b99
+                            stringValue: A good string value
+                            title: Title of abc448ca field
+                            description: Description for abc448ca field
+                            fieldType: string
+      x-lifecycle:
+        status: proposed
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Proposed`
```

Comment:
You have this operation marked as proposed yet the other ones are marked as Trusted Tester. Is there a reason that they are different?

Reply:
No, it's a mistake.